core/templates/dev/head/services/AssetsBackendApiService.ts

```diff
@@ -204,12 +227,24 @@ oppia.factory('AssetsBackendApiService', [
     };
 
     var _getDownloadUrl = function(explorationId, filename, assetType) {
-      return UrlInterpolationService.interpolateUrl(
-        (assetType === ASSET_TYPE_AUDIO ? AUDIO_DOWNLOAD_URL_TEMPLATE :
-        IMAGE_DOWNLOAD_URL_TEMPLATE), {
-          exploration_id: explorationId,
-          filename: filename
-        });
+      if (_initialized) {
```

Comment:
Do you need this clause? I thought the "if _initialized" check in _initialize() should take care of it seamlessly. It would be better to drop lines 230-238 as long as you verify the code is still correct.